workflows/hcp_pnl_topup.lsf

```diff
@@ -39,6 +39,7 @@ COMMENT
 bids_data_dir=/data/pnl/U01_HCP_Psychosis/data_processing/BIDS/rawdata
 
 # write four templates in the following order
+# change sessions to be precise in template instead of using sessionlist with * in template
```

Comment:
In this script, do we ever use `ses-*` ?

Reply:
No, not in this script. I just added the comment because we currently use ses-* in run_gpu_mask.lsf and run_luigi_pnlpipe.lsf templates; it may help clear up confusion for why there is no sessionlist in this script. Feel free to modify or remove the comment if you think it is not necessary. Thanks!